config/routes.rb

```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
   root "polls#index"
 
-  resources :polls
+  resources :polls do
+    resources :candidates
```

Comment:
This isn't needed, as you are using `accepts_nested_attributes_for`, so you can remove this and set it back to just be `resources :polls`